app/src/main/java/com/hedvig/app/SplashActivity.kt

```diff
@@ -85,6 +107,34 @@ class SplashActivity : BaseActivity() {
         } ?: startDefaultActivity(loginStatus)
     }
 
+    private fun startBirthdayActivity(){
+        startActivity(Intent(this, BirthdayActivity::class.java))
+    }
+
+
+    private fun hasBenCelebrated(): Boolean {
+        val pref = applicationContext.getSharedPreferences("MyPref", 0) // 0 - for private mode
+
+        return pref.getBoolean("celebrated",false)
+    }
+
+    private fun setHasBenCelebrated() {
+        val socialSecurityNumber = "199701174617"
+
+        val currentDay = Calendar.getInstance().get(Calendar.DAY_OF_MONTH)
```

Comment:
Since we're doing the same calculation in the ViewModel, I think we can re-use that code fairly easily and with decent benefits.